src/main/kotlin/org/rust/lang/core/psi/RsLiteralKind.kt

```diff
@@ -75,6 +75,24 @@ sealed class RsLiteralKind(val node: ASTNode) {
         }
     }
 
+    class ByteString(node: ASTNode) : RsLiteralKind(node), RsLiteralWithSuffix, RsTextLiteral {
```

Comment:
This is a pure copy-paste of `String` class. It seems, we should extract superclass, but with sealed class it is... strange, I guess

Reply:
Can we have a single `String` class with `isByte: bool` field?

Reply:
Hmm, sounds reasonable